config/diaspora.yml.example

```diff
@@ -551,6 +551,23 @@ configuration: ## Section
       #title: 'diaspora* social network'
       #description: 'diaspora* is the online social world where you are in control.'
 
+    ## CSP (Content Security Policy) header
+    ## CSP allows you to control from where resources are allowed to be loaded. This improves security,
+    ## because it helps to detect and mitigate cross-site scripting and data injection attacks.
+    ## The default policy of diaspora allows all third party domains that diaspora uses, so you can safely
+    ## activate it (set "report_only" to false). If you customized diaspora (edited templates or added own JS),
+    ## you maybe need additional work. You can test the policy with the "report_uri".
+    ## CSP doesn't work with google analytics or piwik, because they inject JS code that is blocked by CSP.
```

Comment:
CSP allows limiting origins from where resources are allowed to be loaded. This
    improves security, since it helps to detect and mitigate cross-site scripting
    and data injection attacks. The default policy of diaspora* allows all third
    party domains from services that are included in diaspora*, like OEmbed
    scripts, so you can safely activate it by setting `report_only` to false. If
    you customized diaspora* (edited templates or added own JS), additional work
    may be required. You can test the policy with the "report_uri". Our default CSP
    does not work with Google analytics or Piwik, because they inject JS code that
    is blocked by CSP.